hack/set-pipeline-vars.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+set -eu -o pipefail
+
+FLY_BIN=${FLY_BIN:-fly}
+PIPELINE_NAME="psn-vpc"
+CLUSTER_NAME="sandbox"
+
+$FLY_BIN -t cd-gsp sync
+
+$FLY_BIN -t cd-gsp set-pipeline -p "${PIPELINE_NAME}" \
+	--config "ci/psn.yaml" \
+  --var "account-name=sandbox" \
+  --var "account-role-arn=arn:aws:iam::011571571136:role/deployer" \
+  --var "cluster-name=${CLUSTER_NAME}" \
+  --var "concourse-pipeline-name=${PIPELINE_NAME}" \
+  --var "concourse-team=gsp" \
+  --yaml-var "github-approvers=[noone]" \
```

Comment:
We should also set this to be a defined list of approvers, once it's live.

Reply:
I think our approach is that this value will be overwritten by the `update` job. Therefore, I think it's safe to keep the default.